### ISSUE TITLE
@objc in separated line

### ISSUE THREAD

**Issue by AntonPalich** [badoo/swift-style-guide#3]
**Preferred:**

```swift
@objc
private func handleNotification() {}
```

**Not Preferred:**

```swift
@objc private func handleNotification() {}
```

**Comment by wiruzx** [comment]
What about `@IBOutlet`?

**Comment by AntonPalich** [comment]
@wiruzx They are mentioned in this rule: https://github.com/badoo/swift-style-guide#access-control. So, `@IBOutlet` should be on the same line.

Why should be an exception for `@objc`? You can provide an Objective-C name in parenthesis, so `@objc` declaration can be really long.

**Comment by wiruzx** [comment]
@AntonPalich I don't see what's the problem of having it on the same line when there is no custom objc name

**Comment by AntonPalich** [comment]
@wiruzx I can't say that it's a problem, but if you use same rule everywhere then you don't need to think were to put `@objc`. Also it's easier to setup tool that checks code style.

**Comment by Viacheslav-Radchenko** [comment]
@wiruzx also when you modify @objc line it wouldn't affect latest blame of method signature

**Comment by radekcieciwa** [comment]
Closed by #18
